### ISSUE TITLE
AA Color Contrast rule fails if an element in a folder is ignored

### ISSUE THREAD

**Issue by henrikfalck** [ygev/sketch-accessibility-assistant#15]
If I ignore any element that sits in a folder (in this instance, a photo) the rule fails to work.

Error message:
```Rule sketch-accessibility-assistant/sketch-accessibility-assistant/aa-contrast-shape produced an error: Rule attempted to report an ignored object in a violation @
report@
n@
@
generatorResume@[native code]
@
Promise@[native code]
@
@
generatorResume@[native code]
@
Promise@[native code]
ue@
@
asyncFunctionResume@[native code]
@[native code]
promiseReactionJobWithoutPromise@[native code]
@[native code]
@
asyncFunctionResume@[native code]
@[native code]
promiseReactionJobWithoutPromise@[native code]
@[native code]
@
asyncFunctionResume@[native code]
@[native code]
promiseReactionJobWithoutPromise@[native code]
@[native code]
@
asyncFunctionResume@[native code]
@[native code]
promiseReactionJobWithoutPromise@[native code]
@[native code]
@
asyncFunctionResume@[native code]
@[native code]
promiseReactionJobWithoutPromise@[native code]
@[native code]
@
asyncFunctionResume@[native code]
@[native code]
promiseReactionJobWithoutPromise@[native code]
@[native code]
@
asyncFunctionResume@[native code]
@[native code]
promiseReactionJobWithoutPromise@[native code]
@[native code]
@
asyncFunctionResume@[native code]
@[native code]
promiseReactionJobWithoutPromise@[native code]
